libs/commons-database/src/main/kotlin/gov/cdc/ocio/database/dynamo/DynamoCollection.kt

```diff
@@ -0,0 +1,126 @@
+package gov.cdc.ocio.database.dynamo
+
+import mu.KotlinLogging
+import gov.cdc.ocio.database.persistence.Collection
+import software.amazon.awssdk.enhanced.dynamodb.*
+import software.amazon.awssdk.services.dynamodb.DynamoDbClient
+import software.amazon.awssdk.services.dynamodb.model.ExecuteStatementRequest
+
+
+/**
+ * DynamoDB collection implementation
+ *
```

Comment:
`@return` statement for Collection is missing in `Kdoc` comments

Reply:
Fixd